blocks/carousel/carousel.js

```diff
@@ -0,0 +1,55 @@
+import {
+  a, div, li, ul, p, img,
+} from '../../scripts/dom-helpers.js';
+
+function apiCardLink(offer) {
+  if ('plucode' in offer) {
+    return `https://www.walgreens.com/store/store/xpo_products.jsp?pluCode=${offer.plucode}`;
+  }
+
+  if ('eventCode' in offer) {
+    return `https://www.walgreens.com/store/BalanceRewardsOffers/balance-rewards-offer.jsp?eventCode=${offer.eventCode}`;
+  }
+
+  // eslint-disable-next-line no-console
+  console.warn('Could not generate link for the following offer', offer);
+  return '';
+}
```

Comment:
I think it would be good to rather than duplicating the code, export the function in the `cards.js` and import it here to be reused.